autoscheduler/frontend/src/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface Course {
+    department: string;
+    courseNumber: number;
+}
```

Comment:
Also should include course titles, ex Data Structures & Algorithms

Reply:
I have added the `courseTitle` field to the `Course` interface, but once we have determined the data format for the backend, the frontend types will likely shadow those closely, and we should be able to fix up `types.ts` after tomorrow's meeting